### PR TITLE
New report: Night battle SS CI for Late Model Torpedo and Submarine Radar

### PR DESCRIPTION
**Pull request by KagamiChan**
We'll have:
+ `shipId`,
+ `CI`: the ci type we guess,
+ `lv`: ship level,
+ `rawLuck`: luck without equipment,
+ `pos`: ship in fleet position, starts from 0,
+ `status`: one of [green, yellow, orange, red],
+ `items`: item ids,
+ `improvement`,
+ `searchLight`: if a ship equips a search light and its hp > 1 at the begining of night battle,
+ `flare`: star shell position, starts from 1,
+ `defenseId`: enemy ship being attacked,
+ `defenseTypeId`: its type,
+ `ciType`: API CI Type,
+ `display`: items displayed in game,
+ `hitType`: hit type,
+ `damage`: damages,
+ `damageTotal`: sum of damages,
+ `time`: the packet timestamp in poi

### CODE REVIEW

File: index.es [hunk 1]
```diff
@@ -623,13 +622,184 @@ class AACIReporter extends BaseReporter {
...
+      setTimeout(() => this.processData(body, time), 1000)
+    } break
+    }
+  }
+}
 
 let reporters = []
 const handleResponse = (e) => {
-  const {method, path, body, postBody} = e.detail
+  const {method, path, body, postBody, time = String(new Date())} = e.detail
```
Comment (yukixz) [inline index.es]: proxy emit timestamp by Date.now(), not 'Tue Sep 19 2017 11:38:23 GMT+0800 (HKT)'

File: index.es [hunk 2]
```diff
@@ -623,13 +622,184 @@ class AACIReporter extends BaseReporter {
...
+  } else {
+    return 'green'
+  }
+}
+
+// T = Torpedo
+// LMT = Late Model Torpedo
+// R = Radar
+
+const getCIType = (equips) => {
```
Comment (yukixz) [inline index.es]: Put private method into Reporter. Dont pollute global scope.

**Review by yukixz — CHANGES_REQUESTED** [review]

**Comment by KagamiChan** [comment]
It should be OK now

**Review by yukixz — APPROVED** [review]

**Comment by yukixz** [comment]
Bump newversion at your free.

**Pull request merged**